images/ansible/extra-vars.yml

```diff
@@ -2,3 +2,6 @@
 offline_token: "{{ lookup('env','REDHAT_OFFLINE_TOKEN') or 'YOUR_TOKEN' }}"
 redhat_username: "{{ lookup('env','REDHAT_USERNAME') or 'YOUR_USERNAME' }}"
 redhat_password: "{{ lookup('env','REDHAT_PASSWORD') or 'YOUR_PASSWORD' }}"
+gcp_service_account_file: "{{ lookup('ansible.builtin.env', 'GCP_SERVICE_ACCOUNT_FILE', default='') }}"
```

Comment:
@cloin FYI I've added more vars to start testing labs using the mesh playbooks. I'll set up some time to learn how to do it if you;re cool with it.